includes/class-wcpdf-main.php

```diff
@@ -1572,7 +1574,40 @@ public function wc_webhook_topics( $topics ) {
 	public function wc_webhook_trigger( $document, $order ) {
 		do_action( "wpo_wcpdf_webhook_order_{$document->slug}_saved", $order->get_id() );
 	}
-	
+
+	/**
+	 * @param string $document_type
+	 * @param \WC_Order|\WC_Order_Refund $order
+	 *
+	 * @return void
+	 */
+	public function display_due_date( string $document_type, $order ): void {
+		if ( 'invoice' !== $document_type ) {
+			return;
+		}
+
+		$invoice_settings = get_option( 'wpo_wcpdf_documents_settings_invoice', array() );
+
+		if ( empty( $invoice_settings['due_date'] ) ) {
+			return;
+		}
+
+		$due_days = ( 'custom' === $invoice_settings['due_date'] ) ? intval( $invoice_settings['due_date_custom_days'] ?? 0 ) : absint( $invoice_settings['due_date'] );
+
+		if ( 0 >= $due_days ) {
+			return;
+		}
+
+		$due_date_datetime = $order->get_date_created()->modify( "+$due_days days" );
```

Comment:
I think we probably need a new option to select which date to use? Because maybe we could also use the Invoice date?